samples/sensor/qdec/README.rst

```diff
@@ -0,0 +1,70 @@
+.. _qdec_sensor:
+
+Quadrature Decoder Sensor
+#########################
+
+Overview
+********
+
+This sample reads the value of the counter which has been configured in
+quadrature decoder mode.
+
+It requires:
+* an external mechanical encoder
+* pin to be properly configured in the device tree
+
+Building and Running
+********************
+
+In order to run this sample you need to:
+
+* enable the quadrature decoder device in your board's DT file or board overlay
+* add a new alias property named ``qdec0`` and make it point to the decoder
+  device you just enabled
+
+For example, here's how the overlay file of an STM32F401 board looks like when
+using decoder from TIM3 through pins PA6 and PA7:
+
+.. code-block:: dts
+
+    / {
+        aliases {
+            qdec0 = &qdec;
+        };
+    };
+
+    &timers3 {
+        status = "okay";
+        st,prescaler = <3>;
+
+        qdec: qdec {
+            status = "okay";
+            pinctrl-0 = <&tim3_ch1_pa6 &tim3_ch2_pa7>;
+            pinctrl-names = "default";
+            input-polarity-inverted;
+            input-filter-level = <15>;
```

Comment:
If an example is given it should match the current bindings.
Could you update the `input-filter-level` to match the actual overlay and add the `st,counts-per-revolution`